FlashLFQ/Bayesian Protein Quantification/ProteinFoldChangeEstimationModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using BayesianEstimation;
+using MathNet.Numerics.Distributions;
+using System.Collections.Generic;
+
+namespace FlashLFQ
+{
+    public class ProteinFoldChangeEstimationModel : Model
+    {
+        public ProteinFoldChangeEstimationModel(double priorMuMean, double priorMuSd, double muInitialGuess,
+            double priorSdStart, double priorSdEnd, double sdInitialGuess, double priorNuExponent, double nuInitialGuess) : base()
+        {
+            modelParameters = new Parameter[3];
+
+            // mu (mean)
+            modelParameters[0] = new Parameter(
```

Comment:
you mean how I decided on those distributions for the priors? the last two are from Kruschke (). 

I used a Student's t-distribution for mu's prior instead of a normal distribution because I wanted long tails. Intuitively, the protein's fold change is likely around some value (the arithmetic mean, the median, zero, etc. depending on assumptions). So it makes sense to use either a normal or Student's t for the prior, because I want the prior probability density to be high around that initial value. I chose Student's t instead of normal because there is a fair amount of uncertainty around what the fold-change actually is. 

In other words, I want it to be likely around a certain value, but it should be plausible to be a different value, especially given enough data. Student's t is good for this purpose.

Reply:
Makes sense. Can you comment that here?